### PR TITLE
Upgrade Quarkus to 2.7.4.Final

### PR DESCRIPTION
**Pull request by zbendhiba**

### CODE REVIEW

**Review by ppalaga — APPROVED** [review]

File: catalog/pom.xml [hunk 1]
```diff
@@ -571,6 +571,19 @@
                 </exclusion>
             </exclusions>
         </dependency>
+        <dependency>
+            <groupId>org.apache.camel.quarkus</groupId>
+            <artifactId>camel-quarkus-azure-servicebus</artifactId>
```
Comment (jamesnetherton) [inline catalog/pom.xml:576]: Please can we do these catalog additions as a separate commit. It makes life easier when cherry-picking stuff over to other branches.

Comment (zbendhiba) [reply catalog/pom.xml:576]: Done !

**Review by jamesnetherton — APPROVED** [review]

**Review by aldettinger — APPROVED** [review]

**Review by zhfeng — APPROVED** [review]

**Comment by zbendhiba** [comment]
The integration test failing on Windows platform is the flaky gRPC Test. So I'd merge this one.
